pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -366,6 +368,30 @@ func (dr *DashboardServiceImpl) SaveDashboard(ctx context.Context, dto *dashboar
 
 	return dash, nil
 }
+func (dr *DashboardServiceImpl) GetSoftDeletedDashboard(ctx context.Context, orgID int64, uid string) (*dashboards.Dashboard, error) {
```

Comment:
iiuc, everyone will be able to see all deleted dashboards because we are not applying any permissions here. is that accurate?

Reply:
This service is similar to `GetDashboard`, and permissions are filtered after calling them.
This is not exposed to the users, the API to retrieve soft deleted dashboards is the `search` and is the same used for regular dashboards using a modifier `deleted=true`.